src/lib/execPromise.ts

```diff
@@ -111,7 +120,12 @@ export const execPromise = ({
         stopSpinner(color.red("failed"));
       }
 
-      rej(new Error(`${printOutput ? "" : stderr}${error}\n${debugInfo}`));
+      res({
+        errorMessage: `${getCommandFailedMessage()}${getErrorMessage(
+          printOutput,
+          stderr
+        )}${error}`,
```

Comment:
same here. Suggest to move message to a separate variable to make it easier to read